scania-digital-design-system/sdds
# feat: removing tegel from sdds

**Describe pull-request**  
Removing the Tegel folder from sdds repo


**How to test**  
1. Check if the ("old") storybook runs OK
2. Check if console trows some error
3. Check if anything from Tegel package/folder remained in sdds repo


**Suggested test steps**

**Additional context**  
_Add any other context about the pull-request here._
